lib/AuthTokenVerifier.ts

```diff
@@ -32,15 +32,15 @@ export class AuthTokenVerifier {
 
     private isAuthorized = async (claimedToken: string): Promise<boolean> => {
         //read publickey field in DID document
-        const didPubKey = this.didDocument.publicKey
-        if (didPubKey.length === 0)
-        return false
+        const didPubKeys = this.didDocument.publicKey
+        if (didPubKeys.length === 0)
+            return false
```

Comment:
If though this is correct, I think it is generally better to allow include braces. It helps avoid bugs in the future. https://github.com/airbnb/javascript/issues/1211
```suggestion
        if (didPubKeys.length === 0) {
            return false
        }
```

Reply:
I don't think you actually need this check either because the `filter` further down will just provide an empty array if `didPubKeys.length === 0`